src/util.js

```diff
@@ -58,7 +58,25 @@ export function chainWith(createDependentObservable) {
   });
 }
 
+/**
+ * @param {string} urlParam  The Url
```

Comment:
better rename the internal variable to a less meaningful name than the param, which is the outside interface of this function and should be kept as meaningful as possible.

eg: urlObject, urlInstance, parsedUrl